CONTRIBUTING.md

```diff
@@ -30,20 +30,20 @@ pnpm start
   [Storybook](https://storybook.js.org/docs/react/get-started/introduction)
   documentation site at http://localhost:6006
 
-### `pnpm` cheat sheet
+### `pnpm` v7 cheat sheet
 
 - `pnpm install` - install the dependencies of every project in the workspace
   and of the workspace itself
-- `pnpm add [-D] <pkg-name> --filter <project-name>` -
+- `pnpm --filter <project-name> add [-D] <pkg-name>` -
```

Comment:
Arg placement needs to be changed only for `pnpm run` no ?

Reply:
I hesitated, but It matches what is recommended in the documentation: https://pnpm.io/filtering -- filtering is a global feature of the CLI, so it makes sense. Also, I think it's good to get into the habit of typing it first to avoid making mistakes with the `run` command.

Reply:
Sensible.